frontend/src/components/Common/FirestoreDataProvider.tsx

```diff
@@ -112,7 +117,10 @@ export default function FirestoreDataProvider({ children }: Props): JSX.Element
         process.env.NODE_ENV === 'test' && children
       }
       {adminEmails == null || members == null || approvedMembers == null ? (
-        <Loader active={true} size="massive" />
+        <Loader style={{ fontSize: 12 }} active={true} size="massive">
+          {!isIDOLMember &&
+            'You may not be registered as a user within IDOL. Please use #idol-support to get assistance.'}
+        </Loader>
```

Comment:
I still see this error message when loading even when signing in with a cornell email. I think this is good to show when its stuck in the infinite loading, but otherwise it just looks like a mistake when users have an actual valid login and still see it.